rdrf/registry/patients/models.py

```diff
@@ -1118,6 +1123,11 @@ def family(self):
         return Family(self)
 
 
+class PatientStage(models.Model):
+    name = models.CharField(max_length=200, null=False)
```

Comment:
`null=False` is the default. Can be omitted.

Reply:
Updated.